### ISSUE TITLE
assets init fails on globally installed cli

### ISSUE THREAD

**Issue by kstaken** [terascope/teraslice#1084]
Trying to use a globally installed version of teraslice-cli. 

```
teraslice-cli assets init standard_assets
execvp(3) failed.: No such file or directory
```

**Comment by kstaken** [comment]
The error above occurs on node:10. On node:8 the error is different.

```
earl assets init standard_assets
yarn run v1.12.3
$ /Users/kstaken/.nvm/versions/node/v8.11.2/lib/node_modules/teraslice-cli/node_modules/.bin/yo --no-insight --no-update-notifier /Users/kstaken/.nvm/versions/node/v8.11.2/lib/node_modules/teraslice-cli/generators/new-asset /Users/kstaken/projects/opensource/assets
module.js:549
    throw err;
    ^

Error: Cannot find module 'yeoman-generator'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/kstaken/.nvm/versions/node/v8.11.2/lib/node_modules/teraslice-cli/generators/new-asset/index.js:5:19)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
All done!
```

**Comment by godber** [comment]
The fundamental problem here is that the lines here:

https://github.com/terascope/teraslice/blob/0bce617d05023c32cde683bf24f4a571273763f6/packages/teraslice-cli/cmds/assets/init.js#L40

**Comment by godber** [comment]
The fundamental problem is that the two lines here:

The fundamental problem here is that the lines here:

https://github.com/terascope/teraslice/blob/0bce617d05023c32cde683bf24f4a571273763f6/packages/teraslice-cli/cmds/assets/init.js#L39-L43

use `yarn yo` ... which is effectively trying to run `./.node_modules/bin/yo` ... or maybe `yo` in your global environment (though it doesn't seem to do that).

I think a solution might be to use https://yeoman.io/authoring/integrating-yeoman.html 

I'm learning about Yeoman basics to understand that.

**Comment by godber** [comment]
Any mechanism that depends on running the external `yo` and/or `yarn` command isn't going to work if we distribute teraslice/cli as a binary `pkg`.

**Comment by godber** [comment]
So, sure, installing `yeoman-generator` works, but it might not be the best choice.

**Comment by godber** [comment]
Actually, it's probably a good idea to add that now so the command isn't DOA.  And I could be wrong about `yo` not working in the `pkg` case, so I'll check that out.

**Comment by jsnoble** [comment]
Has this been resolved through #1083 ?

**Comment by godber** [comment]
Yeah, thanks.
